content/v2.0-preview/composition/composite-resource-definitions.md

```diff
@@ -23,7 +22,7 @@ Crossplane has four core components that users commonly mix up:
 * Composite Resource Definition (`XRD`) - This page. A custom API specification. 
```

Comment:
this list is only 3 items now, but the description setting up this list says "Crossplane has four core components...". This feedback is true for each composition related page that has a similar list.

Reply:
Good catch - this is because I removed claims from this tip box. Will fix.